pynest/examples/compartmental_model/receptors_and_current.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+#
+# receptors_and_current.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Example of a passive three-compartment model that is connected to spike-
+generators via three receptors types and is also connected to a current
+generator.
+"""
+
+import nest
+import matplotlib.pyplot as plt
+
+nest.ResetKernel()
+nest.SetKernelStatus(dict(resolution=.1))
```

Comment:
```suggestion
nest.resolution = 0.1
```
On the other side, you're setting it to the default here, so the line could also be removed...

Reply:
Removed in commits b2f8a3b1e and 050495095